packages/contracts/contracts/interfaces/v0.8.x/IFilteredSharedHolder.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title This interface adds support for including token holder gating when purchasing.
+ * @author Art Blocks Inc.
+ */
+interface IFilteredSharedHolder {
+    event DelegationRegistryUpdated(address delegationRegistry);
+    /**
+     * @notice Registered holders of NFTs at address `_NFTAddress` to be
+     * considered for minting.
+     */
+    event RegisteredNFTAddress(address indexed _NFTAddress);
+    /**
+     * @notice Unregistered holders of NFTs at address `_NFTAddress` to be
+     * considered for minting.
+     */
+    event UnregisteredNFTAddress(address indexed _NFTAddress);
+    /**
+     * @notice Allow holders of NFTs at addresses `_ownedNFTAddresses`, project
+     * IDs `_ownedNFTProjectIds` to mint on project `_projectId`.
+     * `_ownedNFTAddresses` assumed to be aligned with `_ownedNFTProjectIds`.
+     * e.g. Allows holders of project `_ownedNFTProjectIds[0]` on token
+     * contract `_ownedNFTAddresses[0]` to mint.
+     */
```

Comment:
These should be removed now that we no longer are registering/unregistering NFT addresses on-chain